docs/pages/build-reference/infrastructure.mdx

```diff
@@ -68,7 +69,7 @@ Android builders run on virtual machines in an isolated environment. Every build
 
 ### Android server images
 
-#### `ubuntu-22.04-jdk-17-ndk-r26b` (`latest`, `sdk-53`)
+#### <CopyTextButton>`ubuntu-22.04-jdk-17-ndk-r26b` (`latest`, `sdk-53`)</CopyTextButton>
```

Comment:
Would it make sense to have multiple `CopyTextButton`s here so users can pick whether they want to reference the image by its name, `latest` or an `sdk-XX` tag?

Reply:
The goal was just to create a reference to allow copy just the image name.

But yeah, we can add variants to this button component for use cases you mentioned.